build.gradle

```diff
@@ -267,7 +267,7 @@ subprojects { subproject ->
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.16.1'
+        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.18.0_CCD-2779'
```

Comment:
Currently pointing to pre-release, will need to be bumped to 7.18.0 once ccd-test-definitions PR is merged to `master`

Reply:
https://github.com/hmcts/ccd-test-definitions/pull/46